.travis.yml

```diff
@@ -1,14 +1,13 @@
 language: java
-sudo: false
+sudo: required
```

Comment:
I couldn't get our travis builds working w/o sudo.  One of the issues was how chromedriver finds the chrome binary location.  I could revisit the "no-sudo" build with some additional change to support specifying the chrome binary location.